testing_utilities.go

```diff
@@ -97,12 +97,6 @@ func compareFieldNames(t *testing.T, reader1 ion.Reader, reader2 ion.Reader) {
 	// TODO: Add SymbolToken logic here once SymbolTokens are available
 }
 
-func compareNonNullStrings(t *testing.T, str1, str2 string) {
-	assert.NotNil(t, str1, "Expected str1 to be not null")
-	assert.NotNil(t, str2, "Expected str2 to be not null")
-	assert.Equal(t, str1, str2, "Expected strings to match")
-}
-
 func compareAnnotations(t *testing.T, reader1 ion.Reader, reader2 ion.Reader) {
 	// TODO: Add SymbolToken logic here once SymbolTokens are available
```

Comment:
Since you've added ```.``` at the end of other TODO lines, you might want to do this for this line and line 97 as well

Reply:
Added periods to all TODO comments